setup.ps1

```diff
@@ -0,0 +1,229 @@
+# Set the console color to Matrix theme and clear the console
+$host.UI.RawUI.BackgroundColor = "Black"
+$host.UI.RawUI.ForegroundColor = "Green"
+Clear-Host
+
+# Set the log file path and initialize variables
+$LogFilePath = Join-Path $env:TEMP "script_log.txt"
+$ProjectDir = "G:\Downloads\GitHub\freqtrade"
+$RequirementFiles = @("requirements.txt", "requirements-dev.txt", "requirements-hyperopt.txt", "requirements-freqai.txt", "requirements-plot.txt")
```

Comment:
This list is missing `requirements-freqai-rl.txt` ... is that intentional? (torch MAY be hard to install on windows ... :D )

Reply:
No was not intentional, I can add that.